packit/api.py

```diff
@@ -405,7 +405,7 @@ def create_srpm(
         else:
             with self.up.local_project.git_checkout_block(ref=upstream_ref):
                 # upstream repo: create the archive
-                self.up.create_archive(version=upstream_ref)
+                self.up.create_archive(version=version)
```

Comment:
That sound like a deja-vu for me (maybe already in #351?)..;-)

Reply:
yep, that's possible - more rebasing for you, sorry!

Reply:
No, you're fixing the problems one-by-one so in a week, I could close my PR..;-)

Reply:
:D I only fix service bugs